frontend/src/lib/components/PropertyFilters/propertyFilterLogic.js

```diff
@@ -0,0 +1,84 @@
+import { kea } from 'kea'
+import api from '../../../lib/api'
+import { userLogic } from 'scenes/userLogic'
+
+export const propertyFilterLogic = kea({
+    key: props => props.onChange,
```

Comment:
This `key` is not good. The key is converted to a string in the end, as it's stored in redux in an object as a key. `onChange` will always be converted to `[function Function]` or something like that and thus it's void as a unique key.

Reply:
Surprisingly this did fix the issue of navigating between /events and /action/live. I can pass through explicit key but annoyingly that does add another prop (per above comment)

Reply:
Well, technically, it'll add a proper `key`, which is the original intent of props... so I have no issue with that :)